client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import SettingsPaymentsLocationCurrency from './paymentsLocationCurrency';
```

Comment:
We should probably standardize on file name format. All the product files use dashes for separation instead of camel case, and so does Calypso.